modules/core/client/config/core.client.routes.js

```diff
@@ -15,28 +19,52 @@ angular.module('core').config(['$stateProvider', '$urlRouterProvider',
     $stateProvider
     .state('home', {
       url: '/',
-      templateUrl: 'modules/core/client/views/home.client.view.html'
+      views: {
+        'hero': {
+          templateUrl: 'modules/core/client/views/hero.client.view.html'
+        },
+        'main': {
+          controller: 'HomeController',
```

Comment:
@trainerbill I'm pretty sure @rhutchison started that with other PRs, so I don't see a reason why it would be so big in this one.